frontend/src/metabase/collections/utils.ts

```diff
@@ -121,7 +121,11 @@ export function coerceCollectionId(
 export function canonicalCollectionId(
   collectionId: string | number | null | undefined,
```

Comment:
Do we have a scenario in which `collectionId` can be `0`?

Reply:
I don't think so. I think the root collection is always null